xEdit/JvI/xejviScriptAdapter.pas

```diff
@@ -371,6 +395,37 @@ procedure IwbElement_DefType(var Value: Variant; Args: TJvInterpreterArgs);
       Value := Element.ValueDef.DefType;
 end;
 
+procedure IwbElement_DefTypeAsText(var Value: Variant; Args: TJvInterpreterArgs);
+var
+  Element: IwbElement;
+begin
+  Value := '';
+  if Supports(IInterface(Args.Values[0]), IwbElement, Element) then
+    if Assigned(Element.ValueDef) then
```

Comment:
begin
var ValueDef := Element.ValueDef;
If Assigned(ValueDef) then
...
end;

Reply:
Why do you want this change? Is `Element.ValueDef` expensive? Thanks.

Reply:
Given that it's a property with a getter function, in general it can be expensive. 

Primarily it's good to start getting used to this pattern when the same value is used multiple times. With the introduction of inline var statements, doing this has become a lot easier than when you had to define a variable up in the common var block.

Reply:
Okay. Done.